app/models/group.rb

```diff
@@ -92,7 +92,12 @@ def build_repository
                    "(Repository name was: '#{self.repo_name}'). Error message: '#{e.message}'",
                    MarkusLogger::ERROR)
     end
+    true
+  end
 
+  # Set the default repo permissions.
+  def set_repo_permissions
```

Comment:
Assignment Branch Condition size for set_repo_permissions is too high. [17.15/15]